tests/components/ring/conftest.py

```diff
@@ -135,3 +135,16 @@ async def mock_added_config_entry(
     assert await hass.config_entries.async_setup(mock_config_entry.entry_id)
     await hass.async_block_till_done()
     return mock_config_entry
+
+
+@pytest.fixture(autouse=True)
+def mock_ring_event_listener_class():
+    """Fixture to mock the ring event listener."""
+
+    with patch(
+        "homeassistant.components.ring.coordinator.RingEventListener", autospec=True
+    ) as mock_ring_listener:
+        p = PropertyMock()
+        p.return_value = True
+        type(mock_ring_listener.return_value).started = p
```

Comment:
```suggestion
        mock_ring_listener.return_value.started.return_value = True
```

Reply:
This doesn't work:
```
FAILED tests/components/ring/test_light.py::test_updates_work - AttributeError: Mock object has no attribute 'started'. Did you mean: 'start'?
```
Probably because `auto_spec=True` and started is a variable `self.started` in the library

Reply:
oh then you can remove the last `.return_value` I think

Reply:
Yup that worked, many thanks again for your mock expertise!